change/@fluentui-web-components-ec172ca8-23f3-4d01-a969-6504fa67a87a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the FluentUIV0 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33023&commitId=aa88013961b5ae2dc58dd97bb3d2a1d01c3046d1&env=prod&clientType=FLUENTUI)


  <details><summary><b>Box</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Box.BoxExample.chromium.png|12| Changed|
|Box.BoxShorthandExample.chromium.png|12| Changed|

</details>


  <details><summary><b>Checkbox</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.CheckboxExample.chromium.png|51| Changed|
|Checkbox.CheckboxExampleCheckedMixed.chromium.png|72| Changed|
|Checkbox.CheckboxExampleChecked.chromium.png|25| Changed|
|Checkbox.CheckboxExampleCheckedMixedTeamsDark.chromium.png|88| Changed|
|Checkbox.CheckboxExampleToggle.chromium.png|74| Changed|
|Checkbox.CheckboxExampleCheckedMixedTeams.chromium.png|62| Changed|
|Checkbox.CheckboxExampleCheckedMixedTeamsHighContrast.chromium.png|18| Changed|
|Checkbox.CheckboxExampleLabel.chromium.png|22| Changed|
|Checkbox.CheckboxExampleLabelFlexColumn.chromium.png|22| Changed|

</details>


  <details><summary><b>Dropdown</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown.DropdownCustomClearableExampleTeamsDark.chromium.png|639| Changed|
|Dropdown.DropdownExampleLoadingTeamsDark.chromium.png|642| Changed|
|Dropdown.DropdownExampleLoadingTeamsHighContrast.chromium.png|222| Changed|
|Dropdown.DropdownExampleSearchMultipleTeamsDark.chromium.png|642| Changed|
|Dropdown.DropdownExampleTeamsDark.chromium.png|641| Changed|
|Dropdown.DropdownExampleSearchMultipleTeamsHighContrast.chromium.png|222| Changed|

</details>


  <details><summary><b>List</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|List.ListExampleNavigableTeamsDark.chromium.png|1831| Changed|
|List.ListExampleNavigableTeams.chromium.png|117| Changed|
|List.SelectableListControlledExample.chromium.png|121| Changed|
|List.ListExampleNavigableTeamsHighContrast.chromium.png|18749| Changed|
|List.ListExampleSelectable.chromium.png|124| Changed|
|List.ListExampleSelectableTeamsHighContrast.chromium.png|102| Changed|
|List.ListExampleSelectableTeamsDark.chromium.png|2698| Changed|
|List.ListExampleSelectableTeams.chromium.png|117| Changed|
|List.ListExampleNavigable.chromium.png|1428| Changed|

</details>


  <details><summary><b>Menu</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu.MenuExampleDividerHorizontal.chromium.png|76| Changed|

</details>


  <details><summary><b>Reaction</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Reaction.ReactionGroupExample.chromium.png|164| Changed|
|Reaction.ReactionExample.chromium.png|69| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>